src/base1/cockpit.js

```diff
@@ -4113,6 +4107,7 @@ function factory() {
                 channel.removeEventListener("close", on_close);
```

Comment:
for the sake of completeness, should the "control" listener also be removed here?